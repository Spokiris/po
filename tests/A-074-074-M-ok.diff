*** tests/A-074-074-M-ok.out	2023-10-19 17:17:15.000000000 +0100
--- tests/A-074-074-M-ok.outhyp	2023-10-27 13:56:08.211913704 +0100
***************
*** 1,59 ****
- Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Menu de Consultas
- 1 - Procurar valores
- 2 - Procurar funções
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
- Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: 
\ No newline at end of file
--- 0 ----
