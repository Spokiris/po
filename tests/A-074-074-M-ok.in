3
saved26
1
23
34
5
0
4
1
1;1
0
0
