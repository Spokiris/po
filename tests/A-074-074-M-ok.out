Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 3
Ficheiro sem nome. Guardar ficheiro como: saved26
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 1
Especifique o número de linhas da folha: 23
Especifique o número de colunas da folha: 34
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 5
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 4
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1
Especifique a gama (startline;startcol:endline;endcol): 1;1
1;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
